dwave_networkx/drawing/zephyr_layout.py

```diff
@@ -78,7 +79,24 @@ def zephyr_layout(G, scale=1., center=None, dim=2):
         t = G.graph.get('tile')
         coord = zephyr_coordinates(m, t)
         pos = {v: xy_coords(*coord.linear_to_zephyr(v)) for v in G.nodes()}
+        
+    if center is None:
+        center = np.zeros(dim)
+    else:
+        center = np.asarray(center)
 
+    pos_arr = np.array([(pos[v]-center)[:2] for v in G.nodes()])
+    min_x, min_y = np.min(pos_arr, axis=0)
+    max_x, max_y = np.max(pos_arr, axis=0)
+    scale_x = max_x - min_x
+    scale_y = max_y - min_y
+    pos_arr = pos_arr - np.array([min_x , max_y]) # shift to make (0, 0) the top left corner
+    pos_arr = pos_arr * np.array([1/scale_x, 1/scale_y]) # scale to make (1, -1) the bottom right corner
+    paddims = dim - 2
+    zeros = np.zeros((pos_arr.shape[0], paddims))
+    pos_arr = np.hstack((pos_arr*np.array([scale, scale]), zeros)) + center 
+    pos = {v: pos_arr[i] for i, v in enumerate(G.nodes())}
```

Comment:
Would it make sense to de-dup this code by moving it into an utility function?

Reply:
Do you mean to use it in both `pegasus_layout.py` and `zephyr_layout.py`? It's unlikely it gets repeated elsewhere. But will do if you see fit for these two files. Your call.

Reply:
Since you don't see this code used elsewhere, no need to extract it into a utility function. That would be a premature optimization.